contracts/src/v0.8/functions/dev/1_0_0/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -48,19 +49,19 @@ contract TermsOfServiceAllowList is Routable, ITermsOfServiceAllowList, IAccessC
    * @notice Sets the configuration
    * @param config bytes of config data to set the following:
    *  - enabled: boolean representing if the allow list is active, when disabled all usage will be allowed
-   *  - proofSignerPublicKey: public key of the signer of the proof
+   *  - signerPublicKey: public key of the signer of the proof
    */
   function _updateConfig(bytes memory config) internal override {
-    (bool enabled, address proofSignerPublicKey) = abi.decode(config, (bool, address));
-    s_config = Config({enabled: enabled, proofSignerPublicKey: proofSignerPublicKey});
+    (bool enabled, address signerPublicKey) = abi.decode(config, (bool, address));
+    s_config = Config({enabled: enabled, signerPublicKey: signerPublicKey});
     emit ConfigSet(enabled);
   }
 
   /**
    * @inheritdoc ITypeAndVersion
    */
   function typeAndVersion() public pure override returns (string memory) {
```

Comment:
nit: i prefer the 

  string public constant override typeAndVersion = "EVM2EVMOnRamp 1.0.0";

syntax over a "real" function

Reply:
Neat. I like that more too.